Classes/Service/DemandService.php

```diff
@@ -187,6 +231,91 @@ public function getInnerBoundsForProperty(string $propertyName): array
      */
     protected function getConfiguredDemandProviders(): array
     {
-        // TODO: Initialize DemandProvider objects configured in TypoScript and return in order of execution
+        $configurationManager = GeneralUtility::makeInstance(ConfigurationManager::class);
+        $config = $configurationManager->getConfiguration(ConfigurationManager::CONFIGURATION_TYPE_FULL_TYPOSCRIPT)['config.']['tx_demander.'];
+        $demandProviders = [];
+
+        foreach ($config['demandProviders.'] as $id => $demandProvider)
+        {
+            $demandProviders[$id] = GeneralUtility::makeInstance($demandProvider);
+        }
+        return $demandProviders;
+    }
+
+    /**
+     * Transform necessary values from string to integer.
+     *
+     * @param array $restrictionsArray
+     * @return array
+     */
+    protected function restrictionsToInt(array $restrictionsArray): array
+    {
+        $restrictions = [];
+
+        foreach ($restrictionsArray as $key => $restriction) {
+            if ($restriction['value']) {
+                $value = (is_numeric($restriction['value'])) ? (int)$restriction['value'] : $restriction['value'];
```

Comment:
Numeric could also be a non-integer, but your are casting to integer.

Reply:
The idea is to find all numeric values and cast it to integer, I want to cast '3' to 3.